supervisor/misc/tasks.py

```diff
@@ -131,16 +130,23 @@ async def _update_addons(self):
                 )
                 continue
 
-            # Run Add-on update sequential
-            # avoid issue on slow IO
             _LOGGER.info("Add-on auto update process %s", addon.slug)
-            try:
-                if start_task := await self.sys_addons.update(addon.slug, backup=True):
-                    start_tasks.append(start_task)
-            except AddonsError:
-                _LOGGER.error("Can't auto update Add-on %s", addon.slug)
-
-        await asyncio.gather(*start_tasks)
+            # Call Home Assistant Core to update add-on to make sure that backups
+            # get created through the Home Assistant Core API (categorized correctly).
+            # Ultimately this should be handled by Home Assistant Core itself through
+            # the update entity.
+            entity_id = f"update.{slugify.slugify(addon.name, separator='_')}_update"
```

Comment:
This breaks if users rename the add-on update entities.

To me, it seems rather brittle to call Core here.

What I'd suggest is to extend the `SupervisorAddonUpdateEntity` class in Core instead. We are missing the breaking version info currently, but that can be added. Since we ultimately like to have auto-updating entities on Core end this seems like the right direction to me.